.github/workflows/medic-benchmarks.yml

```diff
@@ -0,0 +1,58 @@
+name: Weekly medic benchmark
+on:
+  schedule:
+    # Runs at 1am every monday https://crontab.guru/#0_1_*_*_1
+    - cron: 0 1 * * 1
+
+jobs:
+  delete-old-benchmarks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - uses: google-github-actions/get-gke-credentials@v0.8.2
+        with:
+          cluster_name: zeebe-cluster
+          location: europe-west1-b
+      - name: Collect benchmark data
+        id: data
+        run: |
+          # Get all namespaces, filter for medic benchmarks and select the oldest one
+          echo "normal=$(kubectl get ns -o name | grep -E namespace/medic-cw-[0-9]+-[a-z0-9]+-benchmark$ | sort -V | head -n1)" >> $GITHUB_OUTPUT
+          echo "mixed=$(kubectl get ns -o name | grep -E namespace/medic-cw-[0-9]+-[a-z0-9]+-benchmark-mixed$ | sort -V | head -n1)" >> $GITHUB_OUTPUT
+      - name: Delete benchmarks
+        run: |
+          kubectl delete ${{ steps.data.normal }}
+          kubectl delete ${{ steps.data.mixed }}
```

Comment:
:x: :?: This is different to our previous approach. We would now not delete the folders from the medic git branch.

Reply:
Yes, that's true. But we are also not adding new stuff to the medic branch. I forgot to mention that in the description. IMO, once it's automated, the medic branch is not needed anymore.

Reply:
Kind of agree, but right now the release benchmarks are also stored in the medic branch afaik.